docs/Parameters.rst

```diff
@@ -1182,6 +1182,49 @@ GPU Parameters
 
    -  **Note**: can be used only in CUDA implementation
 
+CTR Parameters
+--------------
```

Comment:
I can review this more thoroughly tomorrow, but I have a fundamental question...what does "CTR" stand for? Could https://lightgbm.readthedocs.io/en/latest/Advanced-Topics.html#categorical-feature-support or some other section of the documentation be expanded to describe this? If I read this parameter document, I would not know what CTR means, so I wouldn't know what it means to set a prior, for example.

Reply:
cc @shiyu1994